### ISSUE TITLE
Retaining ABC song content in ChordPro output

### ISSUE THREAD

**Issue by gwyndaf** [ChordPro/chordpro#442]
Probably a low-priority enhancement. Could ABC (and potentially other delegate content) be retained in ChordPro output, or determined by config settings?

This portion of my source song:
```
{start_of_abc-keyboard}
scale=0.96
%%singleline 1
X:1
K:C
L:1/8
"C"ce c'e [cc']d' c'e| "C/B"Be c'e [Bc']d' c'e| "F"Fe c'e [Fc']d' c'e| Fe c'e [Fc']d' c'e|

{end_of_abc}
```
Becomes this in ChordPro output, when processed with `keyboard` config (to resolve the conditionality):
```
{start_of_abc}
{image:  align=left scale=0.96}
{end_of_abc}
```
Historically, that probably made sense, if no other software could handle embedded ABC scores.

Now, it looks like MobileSheets can handle ABC embedded in ChordPro, but not (yet) conditional directives. That means I need to re-process my songs so they're instrument-specific, with no conditional directives. That process works well enough (except for #441 ), but does mean I lose my ABC score content.

Right now, it's of limited practical use, but could become so. I mostly use PDFs in MobileSheets, and only use ChordPro files if I need  to use _ad hoc_ transposition, but that doesn't yet seem fully functional for ABC scores.

**Comment by sciurius** [comment]
Yes, ChordPro output is lagging behind... 

I've made a quick fix so that abc and textblock sections are included in the resultant output.

(You'll need `--output=ChordPro --backend-option=msp`)

**Comment by gwyndaf** [comment]
It's good that both are going in the same direction, so a bit of lag not really a big issue.

I figured out that I actually need `--generate=ChordPro  --backend-option variant=msp` (and `--output` for filename).

With that, I can confirm that `_start_of_abc` content is now included in output files.

I noticed a few other effects from using the MSP backend variant, but suspect they arise because of my configs, which are complex and mainly fine-tune layout and typography for PDF output. So, probably best I close this issue as done, and develop a better config for MSP output.
